tests/basic/pure_test.go

```diff
@@ -84,6 +84,9 @@ var _ = Describe("{PureVolumeCRUDWithSDK}", func() {
 		Step("setup credential necessary for cloudsnap", createCloudsnapCredential)
 		contexts = make([]*scheduler.Context, 0)
 
+		err := Inst().V.InitializePureLocalVolumePaths() // Initialize our "baseline" of Pure devices, such as FACD devices or other local FA disks
```

Comment:
Does it make sense to move this into `JustBeforeEach` ?

Reply:
Yes, honestly as well as the cloudsnap credentials.